src/test/test-resize-fs.c

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <linux/magic.h>
+
+#include "missing_magic.h"
+#include "resize-fs.h"
+#include "tests.h"
+
+TEST(resize_fs) {
+        uint64_t *k = NULL;
+        assert_se (resize_fs(2, UINT64_MAX, k) == -ERANGE);
+        assert_se (resize_fs(2, 0, k) == -ERANGE);
```

Comment:
if you set k to NULL anyway, then simply pass the NULL literally as third arg of these calls, the `k` variable is useless in that case...